src/cls/rbd/cls_rbd_types.h

```diff
@@ -248,13 +253,13 @@ struct GroupSnapshotNamespace {
 
   GroupSnapshotNamespace(int64_t _group_pool,
 			 const string &_group_id,
-			 const snapid_t &_snapshot_id) :group_pool(_group_pool),
-							group_id(_group_id),
-							snapshot_id(_snapshot_id) {}
+			 const string &_snapshot_id) :group_pool(_group_pool),
+						      group_id(_group_id),
+						      snapshot_id(_snapshot_id) {}
 
   int64_t group_pool;
   string group_id;
-  snapid_t snapshot_id;
+  string snapshot_id;
```

Comment:
Nit: I think we should either drop the "group_" prefix from all the variables (since it's implied w/ the class name) or be consistent and add it here. I am leaning towards the first choice since it's less verbose.